src/main/python/verification/text_verify.py

```diff
@@ -0,0 +1,155 @@
+        if type(verbose) is not bool:
+            raise TypeError('Bad type for arg verbose - expected bool. Received type ' + str(type(verbose)))
+        # Set minimum number of matches, if zero or less set to one.
+        min_matches = min_matches if min_matches > 0 else 1
+        min_percentage = threshold * 100
+        # Logging for debugging and verbose purposes.
+        logger.debug('Threshold for verification set as: ' + str(min_percentage))
+        logger.debug('Minimum number of matches for verification set as: ' + str(min_matches))
+        logger.debug('Simplified scores to be returned' if not verbose else 'Verbose scores to be returned')
+        logger.debug('Verifying:')
+        # Prettify and log the extracted information.
+        [logger.debug(log_line) for log_line in prettify_json_message(extracted).split('\n')]
+        logger.debug('Against:')
+        # Prettify and log the verifier information.
+        [logger.debug(log_line) for log_line in prettify_json_message(verifier).split('\n')]
+        # Initialise a dictionary to house the final matching percentages.
+        scores = {}
+        # Iterate over the verifier and calculate a percentage match for the values,
+        # if the keys match and the corresponding values exist.
+        for key, value in verifier.items():
+            if key in extracted and extracted[key]:
+                logger.debug('Computing match "' + str(value) + '" and "' + str(extracted[key]) + '"...')
+                scores[key] = self._percentage_match(value, extracted[key])
+                logger.debug('"' + value + '" and "' + extracted[key] + '" match percentage is : ' + str(scores[key]))
+            else:
+                logger.warning('Could not find corresponding field "' + key + '" in extracted information to verify')
+        # Determine the number of scores calculated and initialise a default value for the total match score.
+        num_scores = len(scores)
+        total_score = 0.0
+        # Check if enough matches were found.
+        if num_scores >= min_matches:
+            # Calculate the total match score.
+            total_score = self._total_match(scores)
+        # Either the minimum number of scores criteria was not met.
+        else:
+            logger.warning('A total of ' + str(num_scores) + ' matches were found, which is less than the minimum')
+        # Determine whether or not the text is verified.
+        is_verified = total_score >= min_percentage
+        # Logging for debugging purposes.
+        logger.debug('Intermediate match percentages:')
+        [logger.debug(log_line) for log_line in prettify_json_message(scores).split('\n')]
+        logger.debug('Final match percentage: ' + str(total_score))
+        logger.debug('Threshold to pass: ' + str(min_percentage))
+        logger.debug('Result' + 'Passed' if is_verified else 'Failed')
```

Comment:
Colon and space after `'Result'` so that the logger outputs `Result: Passed` instead of `ResultPassed`